src/entities/async-exports/async-export.ts

```diff
@@ -0,0 +1,266 @@
+import { UniverseEntityOptions, UniverseEntity, EntityFetchOptions } from '../_base'
+import { Universe, UniverseFetchOptions } from '../../universe'
+import { DataExportFilter } from '../data-export/data-export'
+import { BaseError } from '../../errors'
+
+export interface AsyncExportOptions extends UniverseEntityOptions {
+  rawPayload?: AsyncExportRawPayload
+}
+
+export interface AsyncExportResponse {
+  data?: AsyncExportRawPayload[] | AsyncExportRawPayload
+  status: number
+  statusText: string
+}
+
+export interface AsyncExportRawPayload {
+  readonly id?: string
+  readonly created_at?: string
+  readonly updated_at?: string
+  readonly deleted?: boolean
+  readonly active?: boolean
+
+  readonly job_reference_id?: string
+  readonly file_id?: string
+  readonly signed_at?: string
+  readonly expires_at?: string
+
+  readonly type?: string
+  readonly request?: DataExportFilter[]
```

Comment:
Double check with adam that this is the correct payload for things like segments, lists and contact export